frontend/src/components/molecules/BlockLink.vue

```diff
@@ -1,5 +1,5 @@
 ﻿<template>
-	<div v-if="props.hash || props.id" class="inline-block whitespace-nowrap">
+	<div v-if="props.hash" class="inline-block whitespace-nowrap">
```

Comment:
I guess this if statement is always true now:
```suggestion
	<div class="inline-block whitespace-nowrap">
```